packages/react/src/components/helper-text/helper-text.tsx

```diff
@@ -2,7 +2,8 @@ import { c, classy, HelperTextProps as BaseProps, m } from '@onfido/castor';
 import React from 'react';
 
 /**
- * Intended to be used within `Checkbox` and `Radio` components.
+ * Intended to be used within `Checkbox` and `Radio`, or `FieldLabel` (incl. as
+ * children of `Input` and `Textarea`) components.
```

Comment:
This is a little bit confusing.

Does it mean this?

```tsx
<Checkbox | Radio | Input | Textarea | FieldLabel>
  <HelperText />
</>
```

Reply:
Did a little rephrase, have a look now.

Reply:
I like the simplicity.
Doesn't mention the specific components, which for me is an advantage, but is objectively less explicit.